### PR TITLE
v5.22 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
@mkraft For v5.22 changelog, can you help with the following information:

 - Any new config.json settings
 - Database Changes
 - API Changes
 - Websocket Event Changes

**Comment by mkraft** [comment]
> @mkraft For v5.22 changelog, can you help with the following information:
> 
> * Any new config.json settings
> * Database Changes
> * API Changes
> * Websocket Event Changes

# Config
* Adds `ServiceSettings.EnableOpenTracing`
* Adds `ServiceSettings.IdleTimeout`
* Adds `ServiceSettings.ExperimentalChannelSidebarOrganization`
* Adds `Office365Settings` object
* Adds `ExperimentalAuditSettings` object
* Adds `NotificationLogSettings.SMTPServerTimeout`

# Database
* Various new indexes

# API
* Adds `GET api/v4/channels/:channels/moderations`
* Adds `PUT api/v4/channels/:channels/moderations/patch` 
* Adds `GET api/v4/commands`
* Adds `PUT api/v4/commands/move`

# Websocket
* Adds `channel_scheme_updated` websocket message

**Comment by amyblais** [comment]
@mkraft Can you help list out the various new indexes for Database changes?

@jasonblais Please see above list for new config settings for diagnostics docs ^

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,131 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+    - Added ``EnableOpenTracing``, to 
+    - Added ``IdleTimeout``, to 
+    - Added ``ExperimentalChannelSidebarOrganization``, to 
+ - Under ``NotificationLogSettings`` in ``config.json``:
+    - Added ``SMTPServerTimeout``, to 
+ - Added ``Office365Settings`` object to add support for Office365Settings Directory (tenant) Id.
+ - Added ``ExperimentalAuditSettings`` object to
```
Comment (amyblais) [inline source/administration/changelog.md:95]: @wiersgallak @esethna @thefactremains FYI this is the list of config settings that should also be documented at https://docs.mattermost.com/administration/config-settings.html.

**Pull request merged**